engine/lib/navigation.php

```diff
@@ -123,8 +114,8 @@ function elgg_register_menu_item($menu_name, $menu_item) {
  * @return \ElggMenuItem|null
  * @since 1.8.0
  */
-function elgg_unregister_menu_item($menu_name, $item_name) {
-	$menus = _elgg_services()->config->menus;
+function elgg_unregister_menu_item(string $menu_name, string $item_name): ?\ElggMenuItem {
```

Comment:
do we really want this to return a menu item?

Reply:
maybe you wanted to remove the menu item from menu 'a' and register it to menu 'b' this allows that